README.rst

```diff
@@ -72,6 +72,8 @@ Requirements and Tested Platforms
 - Windows (32bit/64bit):
 
  - Windows XP or newer.
+ - We don't support python installed from the windows store due to 
```

Comment:
Python, not python; Windows, not windows; add a link (perhaps the following).

```
`permission errors <https://github.com/pyinstaller/pyinstaller/pull/4702>`_
```

Reply:
Will do